content/publication_resources/bibtex/lahti.bib

```diff
@@ -2,7 +2,7 @@
 @Article{Khalighi2024acm,
   author =	 {Moein Khalighi and Giulio Benedetti and Leo Lahti},
   title =	 {{FdeSolver, a Julia Package for Solving Fractional Differential Equations}},
-  journal =	 {ACM Transactions on Mathematical Software},
+  journal =	 {{ACM Transactions on Mathematical Software}},
```

Comment:
Here I would instead do {{ACM Transactions} on Mathematical Software} because it may depend on journal style how they want to capitalize the latter words. But the first ones, in particular ACM should capital in any case.